be/src/exec/json_scanner.cpp

```diff
@@ -609,6 +614,39 @@ Status JsonReader::_construct_row_without_jsonpath(simdjson::ondemand::object* r
     }
     return Status::OK();
 }
+// process debezium-json record
+Status JsonReader::_construct_row_by_debezium(simdjson::ondemand::object* row, Chunk* chunk) {
+    if (UNLIKELY(_op_col_index < 0)) {
+        return Status::InternalError("Debezium json must apply to PRIMARY KEY table.");
+    }
+    simdjson::ondemand::value op_value, value;
+    // Extract the operation type
+    std::vector<SimpleJsonPath> op_json_paths;
+    RETURN_IF_ERROR(JsonFunctions::parse_json_paths(std::string("$.op"), &op_json_paths));
+    RETURN_IF_ERROR(JsonFunctions::extract_from_object(*row, op_json_paths, &op_value));
+    std::string_view op_sv = op_value.get_string().value();
+
+    std::vector<SimpleJsonPath> row_json_paths;
+    // Based on the operation type, extract the relevant data and set the operation column
+    if (op_sv == "c" || op_sv == "u" || op_sv == "r") {
+        // For create, update, or read operations, use the 'after' field
+        RETURN_IF_ERROR(JsonFunctions::parse_json_paths(std::string("$.after"), &row_json_paths));
+        RETURN_IF_ERROR(JsonFunctions::extract_from_object(*row, row_json_paths, &value));
+    } else if (op_sv == "d") {
+        // For delete operations, use the 'before' field
+        RETURN_IF_ERROR(JsonFunctions::parse_json_paths(std::string("$.before"), &row_json_paths));
+        RETURN_IF_ERROR(JsonFunctions::extract_from_object(*row, row_json_paths, &value));
+    } else {
+        return Status::DataQualityError("Invalid 'op' value in Debezium JSON");
+    }
+
+    // Construct the row with the extracted value
+    simdjson::ondemand::object value_obj;
+    value.get(value_obj);
+    RETURN_IF_ERROR(_construct_row_without_jsonpath(&value_obj, chunk, op_sv == "d"));
```

Comment:
How can a user specify the wanted fields without `json_paths`?

Without specify the `json_paths`, uses can't use `columns`.

Reply:
> How can a user specify the wanted fields without `json_paths`?
> 
> Without specify the `json_paths`, uses can't use `columns`.

In many scenes， users load data are end-to-end same columns from DB to SR, so currently will not support specific column.